core/includes/diff.inc

```diff
@@ -144,16 +144,16 @@ class DiffOpChange extends DiffOp {
  *
  * The algorithm used here is mostly lifted from the perl module
  * Algorithm::Diff (version 1.06) by Ned Konz, which is available at:
- *     http://www.perl.com/CPAN/authors/id/N/NE/NEDKONZ/Algorithm-Diff-1.06.zip
+ * http://www.perl.com/CPAN/authors/id/N/NE/NEDKONZ/Algorithm-Diff-1.06.zip
  *
  * More ideas are taken from:
- *     http://www.ics.uci.edu/~eppstein/161/960229.html
+ * http://www.ics.uci.edu/~eppstein/161/960229.html
  *
- * Some ideas are (and a bit of code) are from from analyze.c, from GNU
- * diffutils-2.7, which can be found at:
- *     ftp://gnudist.gnu.org/pub/gnu/diffutils/diffutils-2.7.tar.gz
+ * Some ideas (and a bit of code) are from analyze.c, from GNU diffutils-2.7,
+ * which can be found at:
+ * ftp://gnudist.gnu.org/pub/gnu/diffutils/diffutils-2.7.tar.gz
  *
- * closingly, some ideas (subdivision by NCHUNKS > 2, and some optimizations)
+ * Finally, some ideas (subdivision by NCHUNKS > 2, and some optimizations)
  * are my own.
```

Comment:
Since we are fixing this, this should either indicate the who the person is and give due credits, or simply say "are original" or something like that.

Reply:
...never mind. There's a list of authors below, which should be enough.

Reply:
Maybe this file should not be changed, as it comes from an [external repository](https://git.wikimedia.org/raw/mediawiki%2fcore.git/HEAD/includes%2fdiff%2fDairikiDiff.php).

Reply:
@kiamlaluno I have been tossing back and forth with this file. Whether to include it or not... But this is written towards the top of the file, so I think that means it's been changed enough that we should be responsible for spelling mistakes in it:

> Backdrop's implementation is directly adopted from https://git.wikimedia.org/raw/mediawiki%2fcore.git/HEAD/includes%2fdiff%2fDairikiDiff.php
> Code has been reformatted to match Backdrop standards, and MediaWiki-specific profiling code has been removed.